src/fastapi_app/routes/api_routes.py

```diff
@@ -0,0 +1,98 @@
+import fastapi
+from fastapi import HTTPException
+from sqlalchemy import select
+
+from fastapi_app.api_models import ChatRequest, ItemPublic, ItemWithDistance, RetrievalResponse
+from fastapi_app.dependencies import ChatClient, CommonDeps, DBSession, EmbeddingsClient
+from fastapi_app.postgres_models import Item
+from fastapi_app.postgres_searcher import PostgresSearcher
+from fastapi_app.rag_advanced import AdvancedRAGChat
+from fastapi_app.rag_simple import SimpleRAGChat
+
+router = fastapi.APIRouter()
+
+
+@router.get("/items/{id}", response_model=ItemPublic)
+async def item_handler(id: int, database_session: DBSession) -> ItemPublic:
```

Comment:
Nit, it'd be nice to be consistent with database_session being either first or last in the list. It's different between these two first routes.

Reply:
Or is that because of path parameters? I am new to Depends

Reply:
nope I just missed it, resolved here https://github.com/Azure-Samples/rag-postgres-openai-python/pull/55/commits/17fc97f8493d49613a3a38d2747ef730234d21ad